plugins/dell/fu-self-test.c

```diff
@@ -103,11 +103,13 @@ fu_plugin_dell_tpm_func (void)
 	g_assert_no_error (error);
 	g_assert (ret);
 
+#ifdef HAVE_GETUID
 	if (tpm_server_running == NULL &&
 	    (getuid () != 0 || geteuid () != 0)) {
 		g_test_skip ("TPM tests require simulated TPM2.0 running or need root access with physical TPM");
 		return;
 	}
+#endif
```

Comment:
I'd think this is asking for a test failure scenario. I would think it should be skipped if not defines getuid.

Reply:
does the plugin make sense at all?

Reply:
At least the way it works now - no, I think it should just not be compiled for !linux for now.